tools/sof-ll-timer-check.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+'''Script to analyze performance data in SOF FW log output'''
+
+import sys
+import re
+import json
+from statistics import median
+
+# allowed error margin before error is raised for
+# lower observed performance (1.05 -> 5% difference
+# required to raise error)
+AVG_ERROR_MARGIN = 1.05
```

Comment:
This feels overkill / YAGNI... why not just bump the thresholds 5%? There are only two of them.

Reply:
@marc-hb I added this on purpose. Basicly anyone setting baseline values is going to run a test and copy values over to the database. Probably we add a option to the test script it self that if you pass "--reference-capture", script will run the test, measure performance, but instead of comparing against expected value, just write the measured values out in jsof/init format, so they can be incorporated in the database (after of course reviewed that values are a sane new baseline).

So to make above possible, it's better the margin is stored elsewhere.